app/services/katello/product_content_importer.rb

```diff
@@ -10,7 +10,7 @@ class ProductContentImporter
     #       "name":"Red Hat Satellite 5.7 (ISOs)",
     #       "vendor":"Red Hat",
     #       "contentUrl":"/content/dist/rhel/server/6/$releasever/$basearch/satellite/5.7/iso",
-    #       "requiredTags":"rhel-6-server",
+    #       "osVersions":"rhel-6",
```

Comment:
I think this was an unintentional sed replace?

Reply:
i love that you assume I used `sed` :laughing:

Reply:
Is there any other way? ;)